.github/actions/setup-nvidia/action.yml

```diff
@@ -9,9 +9,15 @@ inputs:
     type: string
     default: "550.54.15"   # https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-550-54-15/index.html
   GPU_ARCH_TYPE:
-    required: true
+    description: GPU Arch type #"cuda", "cuda-aarch64"
+    required: false
+    type: string
+  ARCH_TYPE:
```

Comment:
How is ARCH_TYPE used? 
I think it is safe to assume setup-nvidia is only called for GPU instances, where if it is not cuda, it must be cuda-aarch64. 
GPU_ARCH_TYPE is enough?